src/openforms/appointments/contrib/qmatic/client.py

```diff
@@ -1,10 +1,48 @@
+from typing import TypedDict
+
 from requests import Session
 
+from .exceptions import QmaticException
 from .models import QmaticConfig
 
+# API DATA DEFINITIONS
+
+
+class ServiceDict(TypedDict):
```

Comment:
Is this ever referenced other than as the base class of `FullServiceDict`?

Reply:
Yeah, the service group `services` is a list of `ServiceDict` which lacks the information of the full service dict. Would've been nice if that returned `FullServiceDict` as then I'd only need to make a single API call but now we need two.